submissions/hannasyn/js-dom/index.js

```diff
@@ -0,0 +1,88 @@
+  },
+  {
+    id: 4,
+    navName: 'Fact number 4',
+    title: 'Calling yourself a feminist opens doors',
+    text: 'Women love feminism. By openly admitting you’re a feminist to your woman shows that you respect her just as much as you respect yourself. Whether you are an interesting or uninteresting man on the planet, if you stand behind feminism you will be awesome and more approachable! Feminism fact.'
+  },
+  {
+    id: 5,
+    navName: 'Fact number 5',
+    title: 'There is nothing that you need to do except think',
+    text: 'No one has the right to tell you what type of feminism you need to be. One of the first steps to feminism is changing the way you think about women in general. Try to picture having a wife with a full-time job and lastly, ask a woman if she wants to be touched by you instead of just doing it.'
+    }
+];
+
+const navList = document.querySelector('.navigation__list');
+const container = document.querySelector('.container');
+
+const paragraph = document.createElement('p');
+const title = document.createElement('h2');
+
+const asideBtn = document.querySelector('.aside-btn');
+const navigation = document.querySelector('.navigation');
+
+title.classList.add('title-content');
+paragraph.classList.add('content');
+paragraph.textContent = 'Feminism is usually misunderstood and is viewed as a man-hating movement But it’s really quite the opposite! Feminism is about establishing equality between both men and women. Many women throughout history have undergone severe sexist laws thus the initiation of feminism movement was created to end discrimination. From eradicating female genital disfigurement to setting “girl power” in the 90’s. Relentless, stimulating, lively, serious, provocative, and above all empowering.';
+container.append(paragraph);
+
+function createNavItem() {
+  facts.forEach(function(fact) {
+    const navItem = document.createElement('li');
+    navItem.classList.add('navigation__list-item');
+    const navLink = document.createElement('a');
+    navLink.classList.add('navigation__list-link');
+    navLink.setAttribute('href', "#");
+    navLink.setAttribute('data-id', fact.id);
+    navLink.innerText = fact.navName;
+    navItem.append(navLink);
+    navList.append(navItem);
+    navLink.current = null;
+    navLink.addEventListener('click', function(e) {
+      e.preventDefault();
+      if (document.querySelector('.navigation__list-item.active')){
+        document.querySelector('.navigation__list-item.active').classList.remove('active');
+      }
+      navItem.classList.add('active');
+      createFact(navLink);
+    });
+  })
+};
+
+function createFact(navLink) {
```

Comment:
First of all, I'd recommend renaming this function because current name is confusing a little bit. Maybe, `displayFact` would be better here.
Also, no need to iterate the whole list of facts. Let's imagine, you clicked to the first link and this function iterated through the whole list of items. Using `Array.prototype.find` could prevent iterating through the facts list and just stops it's work after finding the first suitable item according to the condition.

Reply:
Fixed